Apps/WebClient/src/ClientApp/src/components/private/home/HomeView.vue

```diff
@@ -474,6 +499,35 @@ watch(vaccineRecordState, () => {
                 </p>
             </HgCardComponent>
         </v-col>
+        <v-col
+            v-if="showRecommendationsCardButton"
+            :cols="getGridCols"
+            class="d-flex"
+        >
+            <HgCardComponent
+                data-testid="recommendations-card-btn"
```

Comment:
I always prefer `button` over `btn`.  In this case we can probably remove it altogether since it's a card.

I see a number of other `btn` references added in the dialog.

Reply:
alright it'll be a good thing to consolidate, I'll update to button